tests/spread/build-providers/lxd-try/task.yaml

```diff
@@ -8,16 +8,27 @@ prepare: |
   . "$TOOLS_DIR/snapcraft-yaml.sh"
   set_base "$SNAP_DIR/snap/snapcraft.yaml"
 
+  chown ubuntu "$SNAP_DIR"
```

Comment:
should it be recursive ownership?

Reply:
no, we just need enough ownership to create $SNAP_DIR/prime